biome.json

```diff
@@ -0,0 +1,97 @@
+{
+  "$schema": "https://biomejs.dev/schemas/1.8.3/schema.json",
+  "vcs": {
```

Comment:
biome allows integrating with `git` to read `gitignore` content and processing only staged files. these features are opted in and not enabled by default. if needed they can be enabled.

As of now I have enabled only reading the ignore file with `useIgnoreFile`